meta-lmp-bsp/recipes-bsp/u-boot/u-boot-fio/imx8qm-mek/lmp.cfg

```diff
@@ -29,6 +29,7 @@ CONFIG_CMD_IMPORTENV=y
 CONFIG_CMD_EDITENV=y
 CONFIG_CMD_SAVEENV=y
 CONFIG_BOOTDELAY=-2
+# CONFIG_BOOTCOUNT_LIMIT is not set
```

Comment:
Could you add a sense of unsetting (about moving handling to boot.cmd) to this commit message? Thanks.

Reply:
it there already, no?
```
By default CONFIG_BOOTCOUNT_LIMIT is being set in U-Boot, so explicitly
unset it here.
```

Reply:
The commit is gone :)